### DISCUSSION TITLE
Inheritance from the system theme browser?

### DISCUSSION THREAD

**Discussion by Mike-Ro** [RyanClementsHax/tailwindcss-themer#56] [Q&A] [answered]
Hi!

How can I apply my "my-dark-theme" if the user has a dark color theme specified at the operating system or browser level?

Thanks for the answer!

**Comment by Mike-Ro** [comment, marked as answer]
Solved the issue through the plugin: https://github.com/JNavith/tailwindcss-theme-variants

**Reply by RyanClementsHax** [reply]
I released the ability to do this in the most recent version of this plugin :)
